packages/web/src/common/store/user-list/favorites/sagas.ts

```diff
@@ -47,19 +61,25 @@ const getTrackFavorites = createUserListProvider<Track>({
   getExistingEntity: getTrack,
   extractUserIDSubsetFromEntity: (track: Track) =>
     track.followee_saves.map((r) => r.user_id),
-  fetchAllUsersForEntity: async ({
+  fetchAllUsersForEntity: function* ({
```

Comment:
Oh good question. The short answer is so that the implementations for `fetchAllUsersForEntity` can yield to other generator functions and sagas if necessary. This was needed when I had set these up to use `checkSDKMigration` (which is a generator). Now I'm just using it so that we can fetch the SDK through context instead of passing it in as an injected argument.